src/main/webapp/static/js/web3provider.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+// Unpkg imports
+const Web3Modal = window.Web3Modal.default;
+const WalletConnectProvider = window.WalletConnectProvider.default;
+const Fortmatic = window.Fortmatic;
+const evmChains = window.evmChains;
+
+// Web3modal instance
+let web3Modal;
+
+// Chosen wallet provider given by the dialog window
+let provider;
+
+
+/**
+ * Setup the orchestra
+ */
+function init() {
+
+  console.log("WalletConnectProvider is", WalletConnectProvider);
+  console.log("Fortmatic is", Fortmatic);
+  console.log("window.web3 is", window.web3, "window.ethereum is", window.ethereum);
+
+  // Check that the web page is run in a secure context,
+  // as otherwise MetaMask won't be available
+  if(location.protocol !== 'https:') {
+    // https://ethereum.stackexchange.com/a/62217/620
+    // TODO:
```

Comment:
MetaMask worked for me on `http:`  in a browser

<img width="971" alt="Screen Shot 2022-06-28 at 12 46 04 PM" src="https://user-images.githubusercontent.com/15718174/176106179-caa1aac7-eb90-47a6-b92b-058fc163b024.png">

But not in the MetaMask Android app:

![1656397032313](https://user-images.githubusercontent.com/15718174/176107039-c4c0eeb4-9c96-4471-b9a7-c34da3fdc05a.jpeg)